_posts/2022-04-24-sr2022-knockouts.md

```diff
@@ -0,0 +1,117 @@
+101 matches giving them time to improve their robot and tweak
+their strategy as the league went on. Teams had various designs from {list of
+different designs/strategies}. Particularly notable were {list a few teams that
+stood out to us}. {INCLUDE A FEW PICS OF THESE TEAMS}
+
+## Knockout Matches
+
+The league matches were important as they seeded the knockouts giving a
+potentially easier set of matches to teams placed higher in the league.
+{ANYTHING INTERESTING THAT HAPPENED}
+
+## The Final
+
+After progressing through all the knockout matches, four teams were left to face
+each other in the final: {LIST OF TEAMS}. {DESCRIPTION OF WHAT HAPPENED}. {PIC
+OF FINAL} Results After a close match in the final, it was {1st TEAM} that
+managed to beat out the competition and win the Student Robotics 2022
+competition, closely followed by {2nd TEAM} in second and {3rd TEAM} in third.
+
+The Committee Award is given to the team that displays the most extraordinary
+ingenuity in the design of their robot. As engineers, we appreciate elegance,
+simplicity, and robust engineering. This year, the committee award was given to
+{COMMITTEE TEAM} for their {WHY COMMITTEE AWARD}.
+
+We're always delighted to welcome new teams to Student Robotics and understand
+how big a challenge it can be without prior experience. To recognise this
+additional challenge, we award the Rookie Award to the highest placed newcomer
+in the league which celebrates the incredible achievement they have made. This
+year's recipient was {ROOKIE TEAM} with their {NOTABLE FEATURE} design.
+
+We award the Robot and Team Image Award to the team that presents themselves in
+the most outstanding way. As ever, this year our teams rose to the challenge and
+we saw some fantastic themes! We loved {list a few}, but we decided to give the
+Robot and Team Image award to {TEAM IMAGE} for their {adjective} {TEAM IMAGE
+THEME}. {INSERT GROUP PIC OF TEAM AND ROBOT HERE HERE}
+
+Through social media, teams can share the problems they're facing as well as
+their designs and successes. A few teams stood out to us this year with their
+regular updates on social media, but the winner of the Online Presence Award is
+HAM.
+
+For all the details on the awards we give, check out the [rulebook][rules].
+
+## Thank You
+
+As always, this year competition would not have been possible without all of our
+amazing sponsors. Their generous support allows us to make Student Robotics free
+to enter and help us continue in our mission to bring the excitement of
+engineering and the challenge of coding to young people through robotics.
+
+Each year, a small army of volunteers are responsible for making Student
+Robotics happen. In addition to all those who helped at the main event, we have
+teams of volunteers working throughout the year. Those on our Competition Team
+design, organise and deliver Tech Days and the Competition weekend; our Kit Team
```

Comment:
Miiight be worth a quick explainer of what a Tech Day is (we hope this post will be read by people who don't know).